modules/disable-quickstarts-admin-console.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * web_console/configuring-web-console.adoc
+
+[id="disable-quickstarts-admin-console_{context}"]
+= Disabling quick starts in the web console
+
+You can use the *Administrator* perspective of the web console to disable one or more quick starts.
+
+.Prerequisites
+
+* You have cluster administrator permissions and are logged in to the web console.
+
+.Procedure
+
+. In the *Administrator* perspective, navigate to *Administation* -> *Cluster Settings*.
+. In the *Cluster Settings* page, click the *Configuration* tab.
+. In the *Configuration* page, click the *Console* configuration resource with the description *operator.openshift.io*.
++
+image::cluster-settings-console.png[Image of the Configuration page showing the correct Console configuration resource to select]
+
+. In the *Action* drop-down list, click *Customize*. This takes you to the *Cluster configuration* page.
```

Comment:
```suggestion
. From the *Action* drop-down list, select *Customize*, which opens the *Cluster configuration* page.
```
- Per ISG use "from" and "select" when documenting actions for drop-down lists: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#drop_down_list
- Consider combining the sentences to show cause and effect more directly.